doc/api/assert.md

```diff
@@ -13,6 +13,71 @@ A `strict` and a `legacy` mode exist, while it is recommended to only use
 For more information about the used equality comparisons see
 [MDN's guide on equality comparisons and sameness][mdn-equality-guide].
 
+## Class: AssertionError
+
+A subclass of `Error` that indicates the failure of an assertion.
+All errors thrown by the assert module will be instance of the `AssertionError`
+class.
+
+### new AssertionError(options)
```

Comment:
The same.
Compare: [`new net.Server()`](https://nodejs.org/api/net.html#net_new_net_server_options_connectionlistener) and [`new net.Socket()`](https://nodejs.org/api/net.html#net_new_net_socket_options).